app/views/landing/show.html.erb

```diff
@@ -1,31 +1,88 @@
+<div data-controller="country-selection">
```

Comment:
the indentation at various points in this file are off. non-blocking, but it'd nice to have it cleaned up manually since prettier is giving us trouble.